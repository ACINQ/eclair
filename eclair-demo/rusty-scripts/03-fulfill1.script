# Simple test that we can fulfill (remove) an HTLC after fully settled.
A:offer 1
B:recvoffer
A:commit
B:recvcommit
A:recvrevoke
B:commit
A:recvcommit
B:recvrevoke

B:remove 1
B:commit
A:recvremove
A:recvcommit
B:recvrevoke
A:commit
B:recvcommit
A:recvrevoke

checksync
echo ***A***
A:dump
echo ***B***
B:dump
