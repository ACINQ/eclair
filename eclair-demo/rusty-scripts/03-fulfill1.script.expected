***A***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  SIGNED
REMOTE COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  SIGNED
***B***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  SIGNED
REMOTE COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  SIGNED
